### PR TITLE
Adds a working example of importing and building well known protos.

### PR DESCRIPTION
**Pull request by erickj**
Closes cgrushko/proto_library#10

### CODE REVIEW

File: src/BUILD [hunk 1]
```diff
@@ -32,7 +32,15 @@ cc_proto_library(
...
     name = "person_proto",
     srcs = ["person.proto"],
-    deps = [":address_proto"],
+    deps = [
+        ":address_proto",
+        # Well known protos should be included as deps in the
+        # proto_library rules of the source files importing them.
+        # A list of all @com_google_protobuf well known protos can
+        # seen with:
+        # `bazel query 'visible(//:*, kind(proto_library, @com_google_protobuf//:all))'`
```
Comment (cgrushko) [inline src/BUILD]: Perhaps just `bazel query 'kind(proto_library, @com_google_protobuf//:all)'` ?

**Review by cgrushko — CHANGES_REQUESTED** [review]
Nice! thanks a lot!
I'd like this to reach stackoverflow somehow; do you want to ask the question and immediately answer it?

File: src/BUILD (hunk 1 above)
Comment (erickj) [reply src/BUILD]: done

**Comment by erickj** [comment]
sorry, I'm not used to the github review process... is there an equivalent of [PTAL](https://www.chromium.org/glossary)?

re: stack overflow - I'll try to address that later today when I have a moment and after the commit is integrated so that I can link to it.

**Comment by cgrushko** [comment]
Thanks again for your contribution :)

**Pull request merged**